webapp/TargetedMS/js/TrailingCVPlotHelper.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2016-2018 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+Ext4.define("LABKEY.targetedms.TrailingCVPlotHelper", {
+    extend: 'LABKEY.targetedms.QCPlotHelperBase',
+    statics: {
+        tooltips: {
+            'Trailing CV' : 'Blah '
```

Comment:
@wendyinnis to provide.

Reply:
Still need to swap this with real text. @wendyinnis